# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/encoding/api-basics.any.js.ini
# Commit: 425cbe3c033b
# Full Hash: 425cbe3c033bfc25795979513ebd2bb9fda7338d
# Author: moz-wptsync-bot <wptsync@mozilla.com>
# Date: 2024-11-28 21:54:11
# Regressor Bug: 1933829
# File Overlap Count: 1
# Description:
#   Bug 1933829 - [wpt-sync] Update web-platform-tests to 2d5d222739fdf91743f8cbc58094976a613194e7, a=testonly
#   
#   MANUAL PUSH: wpt sync bot
#   
#   wpt-head: 2d5d222739fdf91743f8cbc58094976a613194e7
# ==============================================================================

diff -r 732bcd7ac971 -r 425cbe3c033b testing/web-platform/meta/encoding/api-basics.any.js.ini
--- a/testing/web-platform/meta/encoding/api-basics.any.js.ini	Wed Nov 27 22:05:28 2024 +0000
+++ b/testing/web-platform/meta/encoding/api-basics.any.js.ini	Wed Nov 27 22:05:56 2024 +0000
@@ -6,10 +6,19 @@
 [api-basics.any.worker.html]
 
 [api-basics.any.shadowrealm-in-sharedworker.html]
-  expected: TIMEOUT
+  expected: ERROR
 
 [api-basics.https.any.shadowrealm-in-serviceworker.html]
-  expected: TIMEOUT
+  expected:
+    if (os == "linux") and not debug and fission and (processor == "x86"): [ERROR, TIMEOUT]
+    if (os == "android") and debug and sessionHistoryInParent: ERROR
+    if (os == "android") and debug and not sessionHistoryInParent: [ERROR, TIMEOUT]
+    if (os == "android") and not debug and sessionHistoryInParent: [ERROR, TIMEOUT]
+    if (os == "android") and not debug and not sessionHistoryInParent: [ERROR, TIMEOUT]
+    if (os == "mac") and debug: [ERROR, TIMEOUT]
+    if (os == "mac") and not debug: [ERROR, TIMEOUT]
+    if os == "win": ERROR
+    [TIMEOUT, ERROR]
 
 [api-basics.any.shadowrealm-in-window.html]
   expected: ERROR
@@ -18,7 +27,9 @@
   expected: ERROR
 
 [api-basics.any.shadowrealm-in-dedicatedworker.html]
-  expected: TIMEOUT
+  expected: ERROR
 
 [api-basics.https.any.shadowrealm-in-audioworklet.html]
-  expected: TIMEOUT
+  expected:
+    if os == "linux": TIMEOUT
+    ERROR